run_benchmark.sh

```diff
@@ -42,43 +58,45 @@ for PREFETCH_SET in "${PREFETCHER_SETTINGS[@]}"; do
-	  if $IS_POWER; then
-      perf stat --big-num -e cache-references,cache-misses,L1-dcache-loads,L1-dcache-load-misses,L1-dcache-stores,L1-dcache-store-misses,L1-dcache-prefetches,L1-dcache-prefetch-misses,LLC-loads,LLC-load-misses,LLC-stores,LLC-store-misses,LLC-prefetches,LLC-prefetch-misses,branches,branch-misses,task-clock,context-switches,cpu-migrations,page-faults,cycles,instructions \
-        --output "$FOLDER/$FILENAME-8bit-stats.txt" \
-        numactl --cpunodebind=$CPUNODE --membind=$MEMNODE benchmark/benchmark --column-count 1 --thread-count "$NTHREADS" --"$CACHE_SET" --data-types 8 > $FOLDER/$FILENAME-8bit.csv
-        "$FOLDER/$FILENAME-8bit-stats.txt"
+  	  FILENAME=benchmark-caching"$CACHE_SET"-prefetch"$PREFETCH_SET"-smt"$SMTLVL"-thread"$NTHREADS"
+      EVENTS="cache-references,cache-misses,branches,branch-misses,task-clock,context-switches,cpu-migrations,page-faults,cycles,instructions"
+  	  if $IS_POWER; then
+        perf stat --big-num -e "EVENTS" \
+          --output "$FOLDER/$FILENAME-8bit-stats.txt" \
+          numactl --cpunodebind=$CPUNODE --membind=$MEMNODE benchmark/benchmark --column-count 1 --thread-count "$NTHREADS" --"$CACHE_SET" --data-types 8 > $FOLDER/$FILENAME-8bit.csv
+          "$FOLDER/$FILENAME-8bit-stats.txt"
 
-      perf stat --big-num -e cache-references,cache-misses,L1-dcache-loads,L1-dcache-load-misses,L1-dcache-stores,L1-dcache-store-misses,L1-dcache-prefetches,L1-dcache-prefetch-misses,LLC-loads,LLC-load-misses,LLC-stores,LLC-store-misses,LLC-prefetches,LLC-prefetch-misses,branches,branch-misses,task-clock,context-switches,cpu-migrations,page-faults,cycles,instructions \
-        --output "$FOLDER/$FILENAME-64bit-stats.txt" \
-        numactl --cpunodebind=$CPUNODE --membind=$MEMNODE benchmark/benchmark --column-count 1 --thread-count "$NTHREADS" --"$CACHE_SET" --data-types 64 > $FOLDER/$FILENAME-64bit.csv
-        "$FOLDER/$FILENAME-64bit-stats.txt"
-    else
-		CPU="${CORE_BINDINGS[i]}"
-		perf stat --big-num -e cache-references,cache-misses,L1-dcache-loads,L1-dcache-load-misses,L1-dcache-stores,L1-dcache-store-misses,L1-dcache-prefetches,L1-dcache-prefetch-misses,LLC-loads,LLC-load-misses,LLC-stores,LLC-store-misses,LLC-prefetches,LLC-prefetch-misses,branches,branch-misses,task-clock,context-switches,cpu-migrations,page-faults,cycles,instructions \
-		    --output "$FOLDER/$FILENAME-8bit-stats.txt" \
-		    numactl --physcpubind=$CPU --membind=$MEMNODE benchmark/benchmark --column-count 1 --thread-count "$NTHREADS" --"$CACHE_SET" --data-types 8 > $FOLDER/$FILENAME-8bit.csv
-		    "$FOLDER/$FILENAME-8bit-stats.txt"
+        perf stat --big-num -e "EVENTS" \
+          --output "$FOLDER/$FILENAME-64bit-stats.txt" \
+          numactl --cpunodebind=$CPUNODE --membind=$MEMNODE benchmark/benchmark --column-count 1 --thread-count "$NTHREADS" --"$CACHE_SET" --data-types 64 > $FOLDER/$FILENAME-64bit.csv
+          "$FOLDER/$FILENAME-64bit-stats.txt"
+      else
```

Comment:
Can you fix the indentation here? `if $IS_POWER; then`, `else` and `fi` are not all on the same level